CHANGELOG.md

```diff
@@ -1,6 +1,46 @@
 # Change Log for rocWMMA
 
-Full documentation for rocWMMA is available at [rocwmma.readthedocs.io](https://rocwmma.readthedocs.io/en/latest/).
+Full documentation for rocWMMA is available at [rocm.docs.amd.com/projects/rocWMMA](https://rocm.docs.amd.com/projects/rocWMMA/en/latest).
+
+## (Unreleased) rocWMMA 1.3.0 for ROCm 6.0.0
+
+### Added
+- Added support for gfx940, gfx941 and gfx942 targets
+- Added support for f8, bf8 and xfloat32 datatypes
+- Added support for HIP_NO_HALF, __ HIP_NO_HALF_CONVERSIONS__ and __ HIP_NO_HALF_OPERATORS__ (e.g. pytorch environment)
```

Comment:
seems an extra whitespace. 
__ HIP_NO_HALF_CONVERSIONS__ and 
__ HIP_NO_HALF_OPERATORS__

Reply:
The markup formatting interprets as bold